src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_local_context.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+import os
+import time
+import mock
+import unittest
+
+from azure.cli.core.parser import IncorrectUsageError
+from azure_devtools.scenario_tests.const import MOCKED_SUBSCRIPTION_ID
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import ScenarioTest, LiveScenarioTest, ResourceGroupPreparer, create_random_name, live_only, record_only
+from azure.cli.core.util import get_file_json
+from knack.util import CLIError
+
+
+class LocalContextScenarioTest(ScenarioTest):
+
+    def test_resource_group_with_local_context(self):
+        self.kwargs.update({
+            'group': 'test_local_context_group',
+            'location': 'eastasia'
+        })
+        self.cmd('local-context on')
```

Comment:
`local-context` or `localcontext`? I use `timeseriesinsights`, same as `appservice`, `managedservices`, etc. We'd better keep this in sync.

Reply:
we confirmed with PM about this, and they prefer `local-context`